src/Network/GRPC/LowLevel/Client.hs

```diff
@@ -54,8 +54,8 @@ data ClientSSLConfig = ClientSSLConfig
 
 -- | Configuration necessary to set up a client.
 
-data ClientConfig = ClientConfig {serverHost :: Host,
-                                  serverPort :: Port,
+data ClientConfig = ClientConfig {clientServerHost :: Host,
```

Comment:
This comment might be moot if we're treating `0.0.0.0` as the "unreleased" version number but I'll make it anyway: the API is changing here and I'd expect to see the version number in the cabal file to indicate that is happening.